tsconfig.prod.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./node_modules/ethereumjs-config/typescript/tsconfig.json",

```

Comment:
Similar comment as above (`tsconfig.json`), can we avoid reaching into `node_modules`?